gcs/CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ## Next
 
+1.  Upgrade Hadoop to 2.10.2/3.2.4.
```

Comment:
any specific reason why we are not upgrading to `3.2.5` in `2.2.x` branch?

Reply:
There hasn't been a 3.2.5 release yet, so I'm guessing you meant to ask about 3.3.5.

I'm keeping within the same major.minor version line, but upgrading to the latest for bug fixes. In theory, an upgrade to 3.3 should be backward-compatible, but I haven't done the full diligence to assess that.